src/gui/torrentcontentmodelfolder.cpp

```diff
@@ -74,7 +75,13 @@ void TorrentContentModelFolder::appendChild(TorrentContentModelItem *item)
     m_childItems.append(item);
     // Update own size
     if (item->itemType() == FileType)
+    {
         increaseSize(item->size());
+    }
+    else if (!m_folderMap.contains(item->name()))
```

Comment:
It is not checked above (where it is added to `m_childItems`), so apparently it is handled elsewhere.